mlflow/genai/utils/data_validation.py

```diff
@@ -0,0 +1,137 @@
+import inspect
+import logging
+from typing import Any, Callable
+
+from mlflow.exceptions import MlflowException
+from mlflow.tracing.provider import trace_disabled
+
+_logger = logging.getLogger(__name__)
+
+
+def check_model_prediction(predict_fn: Callable, sample_input: Any) -> bool:
+    """
+    Validate if the predict function executes properly with the provided input.
+
+    Args:
+        predict_fn: The predict function to be evaluated.
+        sample_input: A sample input to the model.
+    """
+    _logger.info("Testing model prediction with the first sample in the dataset.")
+
+    # Wrap the function to add a decorator for disabling tracing
+    @trace_disabled
+    def _check():
+        predict_fn(**sample_input)
+
+    try:
+        _check()
```

Comment:
```suggestion
        _check()
        return True
```

Is `return missing`?

Reply:
ah sorry the type hint was wrong - this function doesn't need to return